### PR TITLE
Costco in Pickup List

### PR DESCRIPTION
**Pull request by captDaylight**
Add Costco in pickup list if treatment GLP and allowed for the organization.

Screen Shot 2024-02-27 at 2 56 54 PM

### CODE REVIEW

File: apps/patient/src/utils/isGLP.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  'liraglutide',
+  'dulaglutide',
+  'exenatide',
+  'tirzepatide',
+  'lixisenatide',
+  'albiglutide'
+];
+
+export function isGLP(name: string) {
+  const regex = new RegExp(commonGLPs.join('|'), 'i');
```
Comment (samkotlove) [inline apps/patient/src/utils/isGLP.ts:24]: the `i` lowercases right?

Comment (captDaylight) [reply apps/patient/src/utils/isGLP.ts:24]: yup!

Comment (captDaylight) [reply apps/patient/src/utils/isGLP.ts:24]: https://www.tutorialspoint.com/How-to-perform-Case-Insensitive-matching-with-JavaScript-RegExp

File: apps/patient/src/views/Pharmacy.tsx [hunk 2]
```diff
@@ -76,6 +79,19 @@ export const Pharmacy = () => {
 
   const toast = useToast();
 
+  const isMultiRx = flattenedFills.length > 1;
+
+  const isCapsuleTerritory = order?.address?.postalCode in capsuleZipcodeLookup;
+  const enableCourier = !isDemo && isCapsuleTerritory && orgSettings.enableCourierNavigate;
```
[the remark this answers is not in the document]
Comment (jknapp25) [reply apps/patient/src/views/Pharmacy.tsx:85]: sure!

**Comment by jknapp25** [comment]
not sure if this matters, but technically not 24 hrs

image

**Review by jknapp25 — APPROVED** [review]

**Pull request merged**